com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsRendering.cs

```diff
@@ -57,11 +45,7 @@ internal DebugOverdrawMode overdrawMode
         internal int maxPixelCost
         {
             get => m_MaxPixelCost;
-            set
-            {
-                m_MaxPixelCost = value;
-                UpdateDebugSceneOverrideMode();
-            }
+            set => m_MaxPixelCost = value;
         }
```

Comment:
Tiny suggestion :) not sure but I think this + backing field could now be simplified to just `internal int maxPixelCost { get; set; } = 10;`